src/applications/find-forms/components/PdfModal.jsx

```diff
@@ -0,0 +1,76 @@
+import React from 'react';
```

Comment:
We had two separate modals in the code: one for the form search results page and the other for the form detail page. I consolidated those two into this file.

Reply:
Makes sense